### PR TITLE
android apollo3 integration

### PR DESCRIPTION
**Pull request by lbloder**
Docs for https://github.com/getsentry/sentry-java/pull/2109

### CODE REVIEW

File: src/platforms/android/configuration/integrations/apollo3.mdx
Comment (imatwawana) [inline src/platforms/android/configuration/integrations/apollo3.mdx:10]: You shouldn't need this redirect since this is an entirely new page unless there's somewhere you're linking from that's generating this link

```suggestion
```

File: src/platforms/android/configuration/integrations/apollo3.mdx
Comment (imatwawana) [inline src/platforms/android/configuration/integrations/apollo3.mdx:19]: Same edit for Java version

```suggestion
Sentry Apollo3 integration provides the `SentryApollo3Interceptor` and the `SentryApollo3HttpInterceptor`, which create a span for each outgoing HTTP request executed with an [Apollo Kotlin](https://github.com/apollographql/apollo-kotlin) GraphQL client. For easier usage, the integration also provides extension functions on the `ApolloClient.Builder`.
```

File: src/platforms/android/configuration/integrations/apollo3.mdx
Comment (imatwawana) [inline src/platforms/android/configuration/integrations/apollo3.mdx:29]: AP style titles - same edit for Java version

```suggestion
## Configure With Extension
```

File: src/platforms/android/configuration/integrations/apollo3.mdx
Comment (imatwawana) [inline src/platforms/android/configuration/integrations/apollo3.mdx:54]: Same edit for Java version

```suggestion
When using a custom `NetworkTransport`, the `SentryInterceptors` need to be added manually, because `HttpInterceptors` need to be added to the `NetworkTransport`:
```

File: src/platforms/java/common/performance/instrumentation/apollo3.mdx
Comment (imatwawana) [inline src/platforms/java/common/performance/instrumentation/apollo3.mdx:117]: I don't think this needs to be in a note. If you're going to keep it in a note though, add line breaks between the tags and the note text. This is best practice.

File: src/platforms/java/common/performance/instrumentation/apollo3.mdx
Comment (imatwawana) [inline src/platforms/java/common/performance/instrumentation/apollo3.mdx:121]: Avoid passive voice 

```suggestion
To make sure that a coroutine has access to the correct Sentry context, provide an instance of `SentryContext` when launching a coroutine:
```

**Review by imatwawana — COMMENTED** [review]
Made some style and wording edits. Some of the ones I made in the Android file should be replicated in the Java file.

File: src/platforms/java/common/performance/instrumentation/apollo3.mdx
Comment (lbloder) [reply src/platforms/java/common/performance/instrumentation/apollo3.mdx]: Kept it as a note because it was one in the original Apollo Docs. Hope this is ok.

File: src/platforms/android/configuration/integrations/apollo3.mdx
Comment (adinauer) [inline src/platforms/android/configuration/integrations/apollo3.mdx:37]: Is this redundant? It isn't included in the example further down under `## Modify or Drop Spans`

File: src/platforms/java/common/performance/instrumentation/apollo3.mdx
Comment (adinauer) [inline src/platforms/java/common/performance/instrumentation/apollo3.mdx:49]: same here

**Review by adinauer — APPROVED** [review]
Mostly LGTM, some minor changes I'd like to see. Approving now to not have you blocked here whilie I'm gone. Please fix before merging.

**Comment by bruno-garcia** [comment]
@adinauer contractors don't have permission to merge docs, so it's up to us to do it 👌

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**
